### DISCUSSION TITLE
TGCF config error?

### DISCUSSION THREAD

**Discussion by indigo659** [aahnik/tgcf#282] [Q&A] [answered]
I'm trying to get the tgcf bot working but whenever i run tgcf live or tgcf past i get this error

can anybody help out? I've got the .evm file and tgcf.config.yml files in a folder and configured them but i have almost zero coding experience so am out of my depth

thanks

![image](https://user-images.githubusercontent.com/61607260/133167902-3929fa5a-3bb5-4aa1-9f38-a60f8cb354a5.png)

**Comment by tissole** [comment, marked as answer]
It is `.env file`  not `.evm` please correct that if you set the wrong name. The error is caused by an incorrect configuration in the tgcf.config.yml file. Make sure you put the correct values for source and destination. Check this [guide](https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F). Also, see #274 for a template.

**Reply by indigo659** [reply]
yes it is actully env, just a typo in my question i think, I will try and get the tgcf.config.yml file corrected

**Reply by indigo659** [reply]
ok so i've got it working further than that but now i have another problem, would be great if you can help, sorry for the trouble

![image](https://user-images.githubusercontent.com/61607260/133242309-fa6bcb04-a1d9-4de5-8fe4-ee74847f8350.png)

**Reply by tissole** [reply]
This error is caused by the wrong values for API_ID or API_HASH in the env file. Read this [guide ](https://github.com/aahnik/tgcf/wiki/Environment-Variables) and check again if you put the values correctly.
